modules/external-api/src/main/java/org/opencastproject/external/endpoint/BaseEndpoint.java

```diff
@@ -194,6 +194,27 @@ public Response getOrganizationProperties() {
     return RestUtil.R.ok(MediaType.APPLICATION_JSON_TYPE, serializer.toJson(obj(props)));
   }
 
+  @GET
```

Comment:
Being that this is in the external API it would require a new version.  Luckily, OC 9 already has that done, but it's something to keep in mind for next time.

Reply:
Ok, thanks